types/index.d.ts

```diff
@@ -39,12 +39,21 @@ declare module "fastify" {
 }
 
 type FastifyReplyFrom = FastifyPluginCallback<fastifyReplyFrom.FastifyReplyFromOptions>
-
 declare namespace fastifyReplyFrom {
   type QueryStringFunction = (search: string | undefined, reqUrl: string) => string;
+
+  export type customRetryHandler = {
```

Comment:
nit: This isn't really a handler so much as the details, no? 

```suggestion
  export type RetryDetails = {
```

Would need to update the readme too